libmscore/chord.cpp

```diff
@@ -2523,11 +2530,11 @@ Element* Chord::drop(EditData& data)
 
             case ElementType::TREMOLO:
                   {
-                  Tremolo* t = toTremolo(e);
+                  Tremolo* t = static_cast<Tremolo*>(e);
                   if (t->twoNotes()) {
                         Segment* s = segment()->next();
                         while (s) {
-                              if (s->element(track()) && s->element(track())->isChord())
+                              if (s->element(track()) && s->element(track())->type() == ElementType::CHORD)
```

Comment:
Why? `isXxxx() is the new style to be used in master

Reply:
Still an issue.